apps/searchjoin/join.cpp

```diff
@@ -42,6 +42,14 @@
 #ifdef _OPENMP
     #define _GLIBCXX_PARALLEL
     #include <omp.h>
+
+    #if defined(__INTEL_COMPILER)
+        #warning "The Intel Compiler crashes with _GLIBCXX_PARALLEL defined (at least until v16.0.2). Therefore, the feature will be disabled."
```

Comment:
No, this warning would only be issued for future versions of icc and is a clear way to say "Hello future programmer please reevaluate this issue if its still there."